sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert (year / week).evalf(15) == Float(52.17745571428571)
```

Comment:
> It's of minimal value against regressions I think; any regression should be caught by a unit test that was written when the potentially recurring bug was fixed.

Regression tests are not only written when bugs are found. You write regression tests to catch future bugs regardless if you know about the bug or not. It  is especially useful to allow easy refactoring. Whether it is in the immediately tested function or something that is deep down the stack.

BTW, I'm annoyed that I'm arguing back and forth with you about a one line code change. Is this really that big of a deal to you? I'm bowing out of this discussion. You can take it from here.